packages/patterns/src/components/create-pattern-modal.js

```diff
@@ -117,7 +139,12 @@ export default function CreatePatternModal( {
 							{ __( 'Cancel' ) }
 						</Button>
 
-						<Button variant="primary" type="submit">
+						<Button
+							variant="primary"
+							type="submit"
+							disabled={ isSaving }
+							aria-disabled={ isSaving }
+						>
```

Comment:
Should we be using the `isBusy` prop here as well?

Reply:
for sure, thanks, done.